dfastmi/batch/DFastUtils.py

```diff
@@ -132,3 +133,14 @@ def _get_km_bins(km_bin: Tuple[float, float, float], type_characteristic_chainag
     km = km_bin[0] + dx + numpy.arange(lb, ub) * km_step
 
     return km
+
+def get_progloc() -> str:
+    """
+    Get the location of the program.
+
+    Arguments
+    ---------
+    None
+    """
+    progloc = str(pathlib.Path(__file__).parent.parent.absolute())
```

Comment:
```suggestion
    import dfastmi
    progloc = str(pathlib.Path(dfastmi.__file__).parent.absolute())
```

This is more reliable

Reply:
Wow, didn't know this. Will use the last one, this seems the most stable